### ISSUE TITLE
No movement on any axis MKS SBASE

### ISSUE THREAD

**Issue by icthus86** [MarlinFirmware/Marlin#8226]
Motors turn on with no movement using pronterface. I can not move any of all the axis by hand so the steppers are engaging. LCD display is showing where it believes each axis is and changes when told to move.

This could be nothing more than my own stupidity and inexperience!
``
[Configuration(_adv).h.zip](https://github.com/MarlinFirmware/Marlin/files/1439576/Configuration._adv.h.zip)

**Comment by maar1201gh** [comment]
I confirm the same fact. Motors turn on with no movement

**Comment by Spawn32** [comment]
But operates normally with other programs ?

I only use simplify3d so haven't tested with anything else, but works like it should with it, can't see what the difference would be :/

**Comment by Spawn32** [comment]
you could try changing #define MINIMUM_STEPPER_PULSE 4 // (µs) The smallest stepper pulse allowed
in _adv.h

Test it with the double or trippel value, dont know why that would change using different programs, but worth a test :)

**Comment by karabas** [comment]
I connect one motor to X and "Move axis" via display menu. It works as usually. Turns well.
I still did not setup on my printer as Sdsupport is broken for now.

**Comment by icthus86** [comment]
I can't move from the LCD controls or pronterface. I'll have a look at MINIMUM_STEPPER_PULSE later on this afternoon.

**Comment by karabas** [comment]
I use default Marlin2 Sbase settings. Have tried move X via pronterface - works.

**Comment by icthus86** [comment]
@karabas it does work, however I have 1/32 microsteps and 0.9 steppers so I need to change the configuration to match my machine and it just doesn't seem to work.

I have now changed the pulse to 8 and 12 with no changes. I'm assuming this is something to do with my setup being odd?

**Comment by karabas** [comment]
My Sbase jumper  is 1/32 on. Stepper is 1.8. I have not 0.9 steppers to check.

**Comment by icthus86** [comment]
@karabas So have you not set the mircosteps in the configuration_adv.h?

**Comment by Spawn32** [comment]
The examples for Sbase is setup for 1/32 and 1.8 steppers, dont have a 0.9 to test with...

Test with 50us just to check, if that dosent help leave it at puls 4us then, didn't know you had those steppers, there is nothing stopping it for working with 1.8 steppers  but you probably have to figure out the correct config... for 1.8 stepper you probably need to double the steps that are in the sbase example...

**Comment by maar1201gh** [comment]
@Spawn32 Configuration_adv.h has MINIMUM_STEPPER_PULSE 0.
I changed to MINIMUM_STEPPER_PULSE 4 and it work. 
Maybe is better to update Configuration_adv.h
Thanks

**Comment by Spawn32** [comment]
@maar1201gh 
No, you have used the wrong example file, Marlin/src/config/examples/Mks/Sbase/Configuration_adv.h
has MINIMUM_STEPPER_PULSE 4, you should always use the example file if it exists :)

**Comment by icthus86** [comment]
So I ended up starting from scratch again changing one thing at a time and uploading, It all works now and I believe it was my own stupidity! I changed the wrong setting for inverting axis. I used define X_ENABLE_ON 1 which is wrong in my setup and now everything is printing perfectly. A few changes I need to make to the settings such as bed centre and offsets but everything is working well. I was ready for chucking this board as I just couldn't get it to work with smoothieware at all!  

Thank you all!
